README.md

```diff
@@ -144,6 +164,82 @@ Documentation for the flow package can be accessed using standard godoc tool, e.
 godoc github.com/trustmaster/goflow
 ```
 
+## Runtime registration example
```

Comment:
Runtime registration should be available as `Runtime.Register()` method rather than having to hardcode it every time.